src/app/components/content/admin/add-partner/add-partner.component.spec.ts

```diff
@@ -0,0 +1,153 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { AddPartnerComponent } from './add-partner.component';
+import { HttpClientTestingModule, HttpTestingController } from '@angular/common/http/testing';
+import { FormsModule, ReactiveFormsModule } from '@angular/forms';
+import { NO_ERRORS_SCHEMA } from '@angular/core';
+import { AuthenticateService } from '../../../../services/authenticate.service';
+import { User } from '../../../../models/User';
+import { BACK_END_URL } from '../../../../services/questions.service';
+import { Router } from '@angular/router';
+import { RouterTestingModule } from '@angular/router/testing';
+
+describe('AddPartnerComponent', () => {
+  let component: AddPartnerComponent;
+  let service: AuthenticateService;
+  let fixture: ComponentFixture<AddPartnerComponent>;
+  let mockHttpClient: HttpTestingController;
+  let element: HTMLElement;
+
+  const TEST_USER = new User();
+  TEST_USER.emailAddress = 'TestUser@hotmail.com';
+  TEST_USER.password = 'testPassword';
+  TEST_USER.name = 'testName';
+
+  beforeEach(async () => {
+    await TestBed.configureTestingModule({
+      declarations: [ AddPartnerComponent ],
+      imports: [
+        HttpClientTestingModule,
+        RouterTestingModule,
+        FormsModule,
+        ReactiveFormsModule
+      ],
+      schemas: [
+        NO_ERRORS_SCHEMA,
+      ]
+    })
+    .compileComponents();
+  });
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(AddPartnerComponent);
+    component = fixture.componentInstance;
+    service = TestBed.inject(AuthenticateService);
+    element = fixture.debugElement.nativeElement;
+    mockHttpClient = TestBed.inject(HttpTestingController);
+    fixture.detectChanges();
+  });
+
+  it('should create', () => {
+    expect(component).toBeTruthy();
+  });
```

Comment:
This doesn't test really anything, could be removed

Reply:
It test if component can be succesfully made. Just extra test